packages/styles/scss/patterns/sub-patterns/card/index.scss

```diff
@@ -62,6 +75,14 @@
       transition: $duration--moderate-01 motion(standard, productive);
     }
 
+    &--link {
+      height: 100%;
+
+      &:hover {
+        background-color: $hover-ui;
+      }
+    }
+
     &:hover {
       text-decoration: none;
```

Comment:
Should this be moved up under `&--link`? the image is still getting an interaction on static.

Reply:
ah I wasn't sure if we still wanted the image to change on hover regardless of type. Moving it  under `&--link`

Reply:
No, sorry for the confusion.